newIDE/app/src/ObjectEditor/Editors/SpriteEditor/PointsEditor/index.js

```diff
@@ -43,7 +43,7 @@ const horizontalMosaicNodes: EditorMosaicNode = {
   direction: 'row',
   first: 'preview',
   second: 'properties',
-  splitPercentage: 50,
+  splitPercentage: 66.67,
```

Comment:
Does this look fine on all resolutions? (+mobile?)

Reply:
On mobile it use the vertical layout. Even with a 1,000 pixels wide window, it seems ok. It allows a bigger image.